src/lib/svg2component.js

```diff
@@ -11,8 +11,11 @@ module.exports = (name, svg, isTypeScriptOutput) => {
   const $svg = $('svg');
   toReactAttributes($svg, $);
   const children = $svg.html();
-  const viewBox = $svg.attr('viewBox');
-  const iconJsx = `<Icon viewBox="${viewBox}" {...props}>${children}</Icon>`;
+  const attrs = ['viewBox', 'width', 'height'].map(name => {
+    const value = $svg.attr(name);
+    return value ? `${name}="${value}" ` : '';
+  }).join('');
```

Comment:
Are you sure about joining with an empty string? It should work, but it's going to clump everything together, e.g. `viewBox="0 0 1 1"width="1"height="1"`

Reply:
i added extra space instead of join(' ')
```
return value ? `${name}="${value}" ` : '';
```

why?

imagine case when we do not have any of this params 'viewBox', 'width', 'height' - then because of join we will have just 3 empty spaces
<Icon   {...props}>

but with `${name}="${value}" ` and join('') - everything will be fine

Reply:
ah, right, I didn't notice the space